guides/v2.1/cloud/env/environments-start.md

```diff
@@ -132,7 +133,7 @@ To activate an inactive environment, use the `magento-cloud environment:activate
 
 ## Integration environment IP addresses {#ipaddress}
 
-The following table lists incoming and outgoing IP addresses used by {{site.data.var.ece}} [Integration environments]({{ page.baseurl }}/cloud/architecture/pro-architecture.html#cloud-arch-int).These IP addresses are stable, but might change in the future. Prior to any future change, all affected customers will receive ample warning.
+The following table lists incoming and outgoing IP addresses used by {{site.data.var.ece}} [Integration environments]({{ page.baseurl }}/cloud/architecture/pro-architecture.html#cloud-arch-int). These IP addresses are stable, but might change. We will notify all affected customers before making any changes.
```

Comment:
Mixed tense:  ...lists, ...are stable, ...will notify.  Yuck.
Perhaps can say "We notify..."